molecule/signature/playbook.yml

```diff
@@ -4,6 +4,7 @@
   vars:
     - theo_url: https://theo.example.com
     - theo_client_token: zdOPNza4jjtceH5F2rU0iOkIJ2xlV4hGUauKT4cNe8HAp+AMnzYEzSc0EIBGM+MJuqL7gLd6bwIP
+    - theo_agent_hostname_prefix: "test-"
```

Comment:
looks like you are setting prefix here, while the commit comment tells "suffix"...

Reply:
Fixed